pages/workspace/inspector.mdx

```diff
@@ -0,0 +1,232 @@
+
+Press the `Connect to Slack` button and follow the instructions.
+
+Avo Inspector will post new issues it finds in production hourly to the specified slack channel.
+
+## View Events
+
+### Header filters
+
+Here you can choose which environment to examine, the time window and search for event name.
+
+#### Environments
+
+Currently Inspector supports 3 types of environments:
+
+- Development
+- Staging
+- Production
+
+The events are streamed to the Environment specified in the client, based on the environment constant provided when initializing the Avo Inspector SDK.
+
+> Development and Staging events appear in the dashboard in a few minutes after they are fired and Production events can take up to 2 hours.
+
+#### Time window
+
+Currently Inspector supports 4 types of time windows:
+
+- 1 hour
+- 24 hours
+- 7 days
+- 30 days
+
+All the time windows are updated hourly on Production and are close to real time in Development and Staging.
+
+#### Search for event name
+
+Type the vent name in the input field and only events with similar name will be shown.
+
+If you want an exact match put the event name in quotes, i.e. "App Opened".
+
+### Share current view of Inspector
+
+Notice that after you change the header filters the URL also changes. You can share this URL with a collegue and they would see exactly the same slice of inspector data as you filtered.
+
+### Events list
+
+The main part of the View Events screen is the list of all events seen, after the specified filters are applied.
+
+You can expand the events to see the event properties.
+
+If Inspector sees an event that is not yet in your tracking plan there is a shortcut to add it to your Tracking plan.
+
+If Inspector detects an issue it will indicate that in the list.
+
+If you select an event or property row a panel with detailed information will appear on the right. There you would be able to see extended information about the issues and edit the event or property in your tracking plan.
+
+#### Issue Types
+
+- <a href="#missing-usual-property-in-event">Property sometimes missing</a>
+- <a href="#event-property-type-inconsistency">Property type mismatch</a>
+- <a href="#event-is-not-present-on-some-platforms">
+    Event missing on some platforms (coming soon)
+  </a>
+- <a href="#property-missing-on-some-platforms">
+    Property missing on some platforms (coming soon)
+  </a>
+- <a href="#event-volume-change-between-versions">
+    Event volume change significant between versions (coming soon)
+  </a>
+- <a href="#significant-volume-mismatch-between-platforms">
+    Event volume significantly different between platforms (coming soon)
+  </a>
+- <a href="#event-name-casing">Event name casing (coming soon)</a>
+- <a href="#property-name-casing">Property name casing (coming soon)</a>
+- <a href="#global-property-type-inconsistency">
+    Global property type mismatch (coming soon)
+  </a>
+
+##### <a name="missing-usual-property-in-event"></a>Property sometimes missing
```

Comment:
These titles look tiiiny, should we go with #### or some other kind of formatting for this section?

Reply:
I'll extract this into a separate page